app/assets/locales/android_translatable_strings.txt

```diff
@@ -285,7 +285,7 @@ mult.install.bad=The selected ZIP file is not valid. Please choose a valid zip f
 mult.install.progress.baddest=Couldn't write multimedia to the local filesystem at: ${0}
 mult.install.progress.badentry=There was a bad entry in the zip file: ${0}
 mult.install.progress.errormoving=There was a problem copying the multimedia from the zip file, please try again.
-mult.install.progress.invalid.entry=The path of the entry ${0} doesn't match the parent folder, review the content of the ZIP file and try again!
+mult.install.progress.invalid.ccz=The selected CCZ file is invalid, please review its content and try again!

```

Comment:
nit: I like the extra messaging but we probably also don't want to hint that user is expected to be able to inspect content of the ccz - `The selected CCZ file is invalid, please verify it and try again!`

Reply:
True. I approached this with an attacker's persona in mind but let's keep things simple, updating it.